consensus/polybft/statesyncrelayer/state_sync_relayer.go

```diff
@@ -135,7 +135,7 @@ func (r *StateSyncRelayer) AddLog(log *ethgo.Log) {
 
 		for i := startID.Int64(); i <= endID.Int64(); i++ {
```

Comment:
AFAIK, state sync ids cannot be negative numbers (https://github.com/0xPolygon/core-contracts/blob/dev/contracts/root/StateSender.sol#L8), so it is better to treat it as `Uint64` instead:

```suggestion
for i := startID.Uint64(); i <= endID.Uint64(); i++ {
```

Reply:
Done